import java.util.*;

public class gcd4 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int a = sc.nextInt();

        System.out.print("Enter second number: ");
        int b = sc.nextInt();

        while (a != b) {

            if (a > b)
                a = a - b;
            else
                b = b - a;
        }

        System.out.println("GCD = " + a);
    }
}